src/main/java/frc/robot/subsystems/ColorSensor.java

```diff
@@ -8,8 +8,12 @@
 package frc.robot.subsystems;
 
 import edu.wpi.first.wpilibj.TimedRobot;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInWidgets;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardComponent;
```

Comment:
Do we need the shuffleboard imports?

Reply:
Was experimenting to try to get shuffleboard or smartdashboard to display colors but that failed so d e l e t e